addons/mail/static/src/core/web/activity.xml

```diff
@@ -39,7 +39,7 @@
                 </table>
             </div>
             <div t-if="activity.note" class="o-mail-Activity-note text-break" t-out="activity.note"/>
-            <div t-if="activity.mail_template_ids.length > 0" class="o-mail-Activity-mailTemplates">
+            <div t-if="activity.mail_template_ids?.length > 0" class="o-mail-Activity-mailTemplates">
```

Comment:
How comes this become an issue on this PR specifically?
Why not always have it as array?

Reply:
I honestly have no idea, but without the optional chaining, opening the CRM app results in a crash 🤓